api/src/controllers/users.js

```diff
@@ -134,4 +165,17 @@ module.exports = {
       .then(result => res.json(result))
       .catch(err => serverUtils.error(err, req, res));
   },
+
+  info: (req, res) => {
+    let userCtx;
+    try {
+      userCtx = getInfoUserCtx(req);
+    } catch (err) {
+      return serverUtils.error(err, req, res);
+    }
+    return getAllowedDocIds(userCtx).then(docIds => res.json({
+      total_docs: docIds.length,
+      warn: docIds.length >= usersService.DOC_IDS_WARN_LIMIT
```

Comment:
It would be useful to also return the `DOC_IDS_WARN_LIMIT` so we can display this in the UI/CLI so admins will know how close they are to the limit.

Reply:
Done.